src/Microsoft.AspNetCore.Mvc.Formatters.Xml/ProblemDetailsWrapper.cs

```diff
@@ -12,9 +12,11 @@ namespace Microsoft.AspNetCore.Mvc.Formatters.Xml
     /// <summary>
     /// Wrapper class for <see cref="Mvc.ProblemDetails"/> to enable it to be serialized by the xml formatters.
     /// </summary>
-    [XmlRoot(nameof(ProblemDetails))]
+    [XmlRoot("problem", Namespace = Namespace)]
```

Comment:
More of the same with Xml, although it does have more than casing changes:

### 2.1
```xml
<ProblemDetails>
    <Title>Some title</Title>
    <Status>403</Status>
    <Instance>Some instance</Instance>
</ProblemDetails>
```

### 2.2
```xml
<problem xmlns="urn:ietf:rfc:7807">
    <title>Some title</title>
    <status>403</status>
    <instance>Some instance</instance>
</problem>
```